homeassistant/components/deconz/gateway.py

```diff
@@ -158,6 +158,9 @@ def async_add_device_callback(
 
     async def async_update_device_registry(self) -> None:
         """Update device registry."""
+        if self.api.config.mac is None:
+            return None
```

Comment:
`return None` can just be `return` - there's no alternative return value so this is really just an early bail-out on error.

Reply:
I did this because some times I get comments from Martin that I should explicitly return none. But maybe that is only related to the entity state machine 🤔

Reply:
I'm happy to defer to his preference, so this is fine.

Reply:
I PMed him on discord for som clarification about this specific thing and will await his feedback. Thanks @chishm

Reply:
If there's no return value we just `return` if we want to return early. If there's a return value we need to `return None`, explicitly.